.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,62 @@
+<!--
+  Make sure you have read CONTRIBUTING.md completely before you file a new
+  issue! 
+
+  If possible, try to determine if the bug is actually part of the Swift driver,
+  or if the issue is actually from `libmongoc` or `libbson`. If so, you should
+  file the issue with the representative projects.
+-->
+
+## Information
+
+### Swift version
+
+What does this command give you?
+```
+$ swift --version
+```
+
+### Operating system
+
+What does this command give you?
+```
+$ uname -a
+```
+
+### Driver version
+
+What does this command give you?
```

Comment:
if they use cocoapods, we can also ask them to include the podspec here.

Reply:
Done. I have never used Cocoapods, so feel free to critique the addition I've made in response to this.